Source/JavaScriptCore/runtime/JSCPtrTag.h

```diff
@@ -52,6 +51,10 @@ using PtrTag = WTF::PtrTag;
     /* Callee:Native Caller:Native */ \
     v(BytecodePtrTag, PtrTagCalleeType::Native, PtrTagCallerType::Native) \
     v(CustomAccessorPtrTag, PtrTagCalleeType::Native, PtrTagCallerType::Native) \
+    v(GetValueFuncPtrTag, PtrTagCalleeType::Native, PtrTagCallerType::Native) \
+    v(GetValueFuncWithPtrPtrTag, PtrTagCalleeType::Native, PtrTagCallerType::Native) \
+    v(PutValueFuncPtrTag, PtrTagCalleeType::Native, PtrTagCallerType::Native) \
+    v(PutValueFuncWithPtrPtrTag, PtrTagCalleeType::Native, PtrTagCallerType::Native) \
```

Comment:
Don't we call these from JIT code too?

Reply:
I don't think so.  Anything called by the JIT needs to be tagged with OperationPtrTag.  Previously, these were always tagged with CFunctionPtrTag, which should not be called directly from the JIT.